src/main/scala/com/microsoft/partnercatalyst/fortis/spark/transforms/nlp/Tokenizer.scala

```diff
@@ -0,0 +1,9 @@
+package com.microsoft.partnercatalyst.fortis.spark.transforms.nlp
+
+object Tokenizer {
+  @transient private lazy val wordTokenizer = """\b""".r
+
+  def tokenize(sentence: String): Seq[String] = {
```

Comment:
Might as well call this `apply` so we can treat Tokenizer like a global function.